### PR TITLE
[TMA-389] Inital IAAC for Auth

### PR DESCRIPTION
**Pull request by des-hippo**
Draft PR for Auth IAAC. Don't merge until auth are ready to integrate.

### CODE REVIEW

**Comment by des-hippo** [comment]
Added the Auth events to the performance subscription this is now ready for review.

File: event-processing/event-processing-template.yml [hunk 1]
```diff
@@ -1898,6 +2035,7 @@ Resources:
           - AUTH_UPDATE_PROFILE_PHONE_NUMBER
           - AUTH_UPDATE_PROFILE_CONSENT_UPDATED
           - AUTH_UPDATE_PROFILE_TERMS_CONDS_ACCEPTANCE
+          - AUTH_PROFILE_AUTH_APP
```
Comment (phantas) [inline event-processing/event-processing-template.yml:2038]: I counted 56 events implemented in the spreadsheet but only 48 here. If there are missing events, I am happy for  that to be in a separate PR.

**Review by phantas — APPROVED** [review]
Looking at the spreadsheet with the list of events, I think we might be missing events, but I am happy for that to go in a separate PR :+1:

**Comment by phantas** [comment]
The commit history in this PR is also a bit of a mess, and I would have preferred if we had rebased it and cleaned it a bit, but there are no side effects at the end so I am happy that we merge it as is, to unblock Auth
